docker/v1.26.0/ubuntu18.04/Dockerfile

```diff
@@ -0,0 +1,89 @@
+#
+#  Copyright (c) 2024-2025, Intel Corporation
+#
+#  SPDX-License-Identifier: BSD-3-Clause
+
+
+FROM ubuntu:18.04@sha256:152dc042452c496007f07ca9127571cb9c29697f42acbfad72324b2bb2e43c98
+LABEL maintainer="Arina Neshlyaeva <arina.neshlyaeva@intel.com>"
+SHELL ["/bin/bash", "-c"]
+
+ARG REPO=ispc/ispc
+ARG SHA=v1.26.0
+ARG LTO=ON
+ARG PGO=OFF
+ARG XE_DEPS=ON
+
+# !!! Make sure that your docker config provides enough memory to the container,
+# otherwise LLVM build may fail, as it will use all the cores available to container.
+
+RUN uname -a
+
+# Packages required to build ISPC and Clang.
+RUN apt-get -y update && apt-get --no-install-recommends install -y \
+        wget=1.19.4-1ubuntu2.2 build-essential=12.4ubuntu1 gcc=4:7.4.0-1ubuntu2.3 g++=4:7.4.0-1ubuntu2.3 \
+        git=1:2.17.1-1ubuntu0.18 ca-certificates=20230311ubuntu0.18.04.1 ninja-build=1.8.2-1 gnupg=2.2.4-1ubuntu1.6 && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install multilib libc needed to build clang_rt
+RUN if [[ $(uname -m) =~ "x86" ]]; then \
+        export CROSS_LIBS="libc6-dev-i386=2.27-3ubuntu1.6"; \
+    else \
+        export CROSS_LIBS="libc6-dev-armhf-cross=2.27-3ubuntu*"; \
+    fi && \
+    apt-get -y update && apt-get --no-install-recommends install -y "$CROSS_LIBS" && \
+    rm -rf /var/lib/apt/lists/*
+
+# Install TBB (required for ISPCRT).
+RUN apt-get -y update && apt-get --no-install-recommends install -y libtbb-dev=2017~U7-8 && rm -rf /var/lib/apt/lists/*; fi
```

Comment:
```
 => ERROR [ 5/13] RUN apt-get -y update && apt-get --no-install-recommends install -y libtbb-dev=2017~U7-8 && rm -rf /var/lib/apt/lists/*; fi                                                                                                                              0.1s
------
 > [ 5/13] RUN apt-get -y update && apt-get --no-install-recommends install -y libtbb-dev=2017~U7-8 && rm -rf /var/lib/apt/lists/*; fi:
0.125 /bin/bash: -c: line 0: syntax error near unexpected token `fi'
0.125 /bin/bash: -c: line 0: `apt-get -y update && apt-get --no-install-recommends install -y libtbb-dev=2017~U7-8 && rm -rf /var/lib/apt/lists/*; fi'
------
Dockerfile:38
```

I'm trying to build this on ARM.

Reply:
Should be OK now

Reply:
Yep, the typos was fixed. I checked that it builds now on ARM.